pkg/config/config_template.yaml

```diff
@@ -287,6 +287,22 @@ api_key:
 #
 # forwarder_stop_timeout: 2
 
+## @param http_protocol - string - optional - default: auto
+## @env DD_FORWARDER_HTTP_PROTOCOL - string - optional - default: auto
+## The transport type to use for sending logs. Possible values are "auto" or "http1".
+# forwarder_http_protocol: auto
+
+## @param forwarder_max_concurrent_requests - integer - optional - default: 10
+## @ENV DD_FORWARDER_MAX_CONCURRENT_REQUESTS - integer - optional - default: 10
+## The maximum number of concurrent requests that each worker can have queued up
+## at any one time. If the connection is over HTTP/1 each request will be waiting
+## for the previous request to complete before sending the next one. With HTTP/2
+## each request can be sent before waiting for the response.
+##
+## A value of 0 means there is no limit applied.
+#
+# forwarder_max_concurrent_requests: 10
```

Comment:
How was the default of `10` arrived at?

Reply:
It was largely arbitrary. The default maximum in HTTP/2 is 100, but I feel this is too much for HTTP/1 connections since the goprocs would just pile up. So I have chosen a conservative value of 10. Benchmarking has shown that it still provides a great improvement, but it's likely something we will want to tune as HTTP/2 rolls out more.